worldengine/model/world.py

```diff
@@ -852,7 +851,67 @@ def is_chaparral(self, pos):
     #
 
     def n_actual_plates(self):
-        return self.layers['plates'].data.max() + 1
+        return self.plates.data.max() + 1
+
+    #
+    # Accessors
+    #
+
+    @property
+    def elevation(self):
+        return self.layers['elevation']
```

Comment:
I have not done much with `@property` myself so far, so maybe this is a stupid question: Is there no need for an explicit `set()`-function? Will Python create one on its own? [The properties are currently set via their own function](https://github.com/Mindwerks/worldengine/blob/issue209/worldengine/simulations/irrigation.py#L11), so the whole "property"-idea isn't fully made use of in my opinion.
```python
@property
def elevation(self):
    return self.layers['elevation'].data

@elevation.setter
def elevation(self, value):
    self.layers['elevation'].data = value
```
That second part is missing.